_posts/2024-09-26-managing-connector-offsets.md

```diff
@@ -0,0 +1,252 @@
+---
+layout: post
+title:  "Managing Connector Offsets"
+date: 2024-08-26
+author: kate_stanley
+---
+
+When building data pipelines using Kafka Connect, or replicating data using MirrorMaker, offsets are used to keep track of the flow of data.
+Sink connectors use the usual Kafka consumer offset mechanism, while source connectors can store offsets with a custom format in a Kafka topic.
+
+To manage connector offsets, rather than directly interacting with the underlying Kafka topics, you can make use of the following endpoints from the Connect REST API:
+
+* `GET /connectors/{connector}/offsets` to list offsets
+* `PATCH /connectors/{connector}/offsets` to alter offsets
+* `DELETE /connectors/{connector}/offsets` to reset offsets
+
+In the next release of Strimzi you will also be able to manage connector offsets directly in the KafkaConnector and KafkaMirrorMaker2 custom resources.
```

Comment:
Maybe? 
```suggestion
In the next release of Strimzi (i.e., 0.44.x) you will also be able to manage connector offsets directly in the KafkaConnector and KafkaMirrorMaker2 custom resources.
```

Reply:
I've updated the wording to similar to what you suggested